### ISSUE TITLE
Fonts: OUTLINE seems too thick

### ISSUE THREAD

**Issue by mrbuds** [Stanzilla/WoWUIBugs#286]
On 10.0 `OUTLINE` flag on fonts adds a border as large as `THICKOUTLINE`, only difference seems to be in the spacing between letters

```Lua
local frame = CreateFrame("Frame")
local text = frame:CreateFontString()
text:SetFont("Fonts\\FRIZQT__.TTF", 16, "")
text:SetText("NONE")
text:SetPoint("TOP", UIParent, "TOP", 0, -20)


local text2 = frame:CreateFontString()
text2:SetFont("Fonts\\FRIZQT__.TTF", 16, "OUTLINE")
text2:SetText("OUTLINE")
text2:SetPoint("TOP", UIParent, "TOP", 0, -50)

local text3 = frame:CreateFontString()
text3:SetFont("Fonts\\FRIZQT__.TTF", 16, "THICKOUTLINE")
text3:SetText("THICKOUTLINE")
text3:SetPoint("TOP", UIParent, "TOP", 0, -80)
```

Dragonflight: 10.0 BETA
![image](https://user-images.githubusercontent.com/189656/192175771-cee94208-2639-4124-ae86-c724d749011f.png)

Dragonflight pre-patch 10.0 PTR
![image](https://user-images.githubusercontent.com/189656/192176262-b2c2c4fe-67f1-40b4-b37a-bd00f580eb19.png)

Shadowlands:
![image](https://user-images.githubusercontent.com/189656/192175883-ab4454f3-05a3-425c-ab56-5b45ef5703d5.png)

**Comment by EKE00372** [comment]
slug issue i think......
.slug is none
.slugo is outline, it actually only have one outline style

**Comment by CyanoHao** [comment]
It seems to be an issue of the [Slug font library](https://sluglibrary.com/).

In short, Slug library is a GPU-based font rendering library, which is highly different from CPU-base font rendering, and thus Slug has its own font format. For outlined (either normal outline or thick outline) version of each font, WoW generates one slug cache file (`Fonts/<FileID>.slugo`) for Slug library.

From chapter 7 (Font Conversion) of [Slug user manual](https://sluglibrary.com/SlugManual.pdf), there exists an option `-outline <size>` to set the outline size. I’m not sure why WoW doesn’t generate 2 cache files for normal outline and thick outline respectively.

Besides, if replace the default font with a large (in file size and in character set) font (e.g. [Source Han Serif](https://github.com/adobe-fonts/source-han-serif/blob/2.001R/OTF/SimplifiedChinese/SourceHanSerifSC-Regular.otf)), the outline disappears!

![image](https://user-images.githubusercontent.com/14287996/192183138-b454fe3e-d7f1-47bb-84a0-c4957c6d927b.png)

**Comment by Meorawr** [comment]
Can confirm this issue is slug related - `/console UseSlug 0` makes the given test case display as expected.

**Comment by Meorawr** [comment]
Appears to be resolved in 10.0.2.45969.
